### PR TITLE
Homework27: Pavel Mitrafanau

### PR DESCRIPTION
**Pull request by Pavel2812mi**
Good day, Sergei
My API test-cases are ready for review
For some reason i get 404 {'error': 'Order not found'} error when try to delete user

### CODE REVIEW

File: homework27/Requests.py
Comment (Stml89) [inline homework27/Requests.py:1]: the same module name as the installed one, please re-name the module

File: homework27/Requests.py [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+    Args:
+        updated_user_id (str): The ID of the updated user to delete.
+    """
+    logger.info("Starting a test: Delete updated user")
+    response = requests.delete(f"{Config.base_url}"
+                               f"/functions/deleteUser/"
+                               f"{updated_user_id}",
+                               headers=Config.get_auth_headers(), timeout=5)
+    print(response.status_code)
```
Comment (Stml89) [inline homework27/Requests.py]: Use `logger` instead of `print`

File: homework27/Requests.py
Comment (Stml89) [reply homework27/Requests.py:1]: https://peps.python.org/pep-0008/#package-and-module-names

File: homework27/Test_data.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+name = fake.name()
+email = fake.email()
+age = fake.random_int(min=18, max=99)
+
+
+def generate_user_phone_number():
+    """generates phone number in correct format"""
+    msisdn = fake.msisdn()
+    formatted_phone_number = '+' + msisdn
+    return formatted_phone_number
```
Comment (Stml89) [inline homework27/Test_data.py]:
```suggestion
msisdn = '+' + fake.msisdn()
```

File: homework27/Test_data.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+phone_number = generate_user_phone_number()
```
Comment (Stml89) [inline homework27/Test_data.py]:
```suggestion

```

File: homework27/Test_data.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+address = generate_user_address(10)
+
+
+def generate_user_role():
+    """generates user role"""
+    roles = ["user", "admin", "moderator"]
+    return random.choice(roles)
+
+
+role = generate_user_role()
```
Comment (Stml89) [inline homework27/Test_data.py]:
```suggestion
role = random.choice(["user", "admin", "moderator"])
```

File: homework27/Test_data.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+def generate_user_address(min_length=10):
+    """generates address in correct format"""
+    generated_address = fake.address()
+    while len(generated_address) < min_length:
+        generated_address = f"{generated_address}, {fake.city()}"
+    return generated_address
+
+
+address = generate_user_address(10)
```
Comment (Stml89) [inline homework27/Test_data.py]: do not execute function, import and use where you need it

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: homework27/Requests.py
Comment (Pavel2812mi) [reply homework27/Requests.py:1]: Renamed module

File: homework27/Requests.py (hunk 1 above)
Comment (Pavel2812mi) [reply homework27/Requests.py]: Done

File: homework27/Test_data.py (hunk 2 above)
Comment (Pavel2812mi) [reply homework27/Test_data.py]: Done

File: homework27/Test_data.py (hunk 3 above)
Comment (Pavel2812mi) [reply homework27/Test_data.py]: Done

File: homework27/Test_data.py (hunk 4 above)
Comment (Pavel2812mi) [reply homework27/Test_data.py]: Done

File: homework27/Test_data.py (hunk 5 above)
Comment (Pavel2812mi) [reply homework27/Test_data.py]: Done

**Review by Stml89 — APPROVED** [review]

**Pull request merged**